app/styles/ui/_stash-diff-viewer.scss

```diff
@@ -6,23 +6,50 @@
   flex: 1;
 
   .header {
+    display: flex;
+    flex-direction: column;
+    border-bottom: var(--base-border);
+    padding: var(--spacing-triple) var(--spacing-double);
+
     h3 {
       font-weight: var(--font-weight-light);
       font-size: var(--font-size-xl);
       line-height: 1;
       margin-bottom: var(--spacing);
     }
-    display: flex;
-    flex-direction: column;
-    border-bottom: var(--base-border);
-    padding: var(--spacing-double);
+
+    .row {
+      display: flex;
+      align-items: center;
+      margin-top: var(--spacing);
+    }
 
     .button-group {
       display: flex;
       button {
         margin-right: var(--spacing);
       }
     }
+
+    .explanatory-text {
+      display: flex;
+      align-items: center;
+      flex-grow: 1;
+      margin-left: var(--spacing-half);
+
+      .text {
+        flex-grow: 1;
+      }
+
+      .octicon {
+        fill: var(--text-secondary-color);
+        margin: 0 var(--spacing-half);
+      }
+
+      .octicon {
+        flex-shrink: 0;
```

Comment:
Could we move this up into the previous `.octicon` selector?

Reply:
lol yeah that's a very reasonable request! thanks for catching that. 73c86d6